### DISCUSSION TITLE
文档中"provider:geosite:cn@attr@attr,us@attr..." 这句里 “@”的用法和v2ray 里一样吗？

### DISCUSSION THREAD

**Discussion by wy16W2pIilK1xgqN** [IrineSistiana/mosdns#477] [Q&A] [answered]
```
# 也可以是 v2ray 域名数据。多个标签用 "," 分割，标签后支持用 "@" 只载入有对应 attr 的数据。
# 多个 "@" 为"或"关系，域名只要有其中任意 attr 就会被载入。
- "provider:geosite:cn@attr@attr,us@attr..."
```
文档中的这句里， "标签后支持用 "@" 只载入有对应 attr 的数据。"   @ 的用法有些不理解。
是不是和  v2ray-rules-dat 中的用法一样？
请大家指点一下。
*^____^*

**Comment by IrineSistiana** [comment, marked as answer]
v2ray-rules-dat 只是个数据包。里面每个条目可以有一个或多个 attr 标识。具体怎么解读/过滤/这个标识取决于程序。

你要是问和 v2ray 的语法是不是一样。答：不一样。v2ray 不支持多个tag连用而且不支持多个attr筛选。
